bl32/sp_min/sp_min_main.c

```diff
@@ -207,3 +207,25 @@ void sp_min_warm_boot(void)
 	copy_cpu_ctx_to_smc_stx(get_regs_ctx(cm_get_context(NON_SECURE)),
 			next_smc_ctx);
 }
+
+#if SP_MIN_WITH_SECURE_FIQ
+/******************************************************************************
+ * This function is invoked on secure interrupts. By construction of the
+ * SP_MIN, secure interrupts can only be handled when core executes in non
+ * secure state.
+ *****************************************************************************/
+void sp_min_fiq(void)
+{
+	uint32_t id;
+
+	id = plat_ic_acknowledge_interrupt();
+	plat_sp_min_interrupt_handler(id);
+	plat_ic_end_of_interrupt(id);
+}
+
+#pragma weak plat_sp_min_interrupt_handler
+void plat_sp_min_interrupt_handler(uint32_t id)
+{
+	VERBOSE("[sp_min] secure interrupt %d\n", (int)id);
+}
```

Comment:
I don't see the value in this weak definition. The feature has no value unless the platform provides a useful implementation. I suggest removing to force the platform to provide an implementation.